revscoring/dependent.py

```diff
@@ -12,9 +12,11 @@ def __init__(self, name, process, dependencies=None):
         self.name = name
         self.process = process
         self.dependencies = dependencies if dependencies is not None else []
+        self.calls = 0
     
     def __call__(self, *args, **kwargs):
         logger.debug("Executing {0}.".format(self))
+        self.calls += 1
```

Comment:
Indeed, that is possible, but Python gracefully transitions to using longs when necessary.

See http://stackoverflow.com/questions/13795758/what-is-sys-maxint-in-python-3